apps/camera_pipe/Makefile

```diff
@@ -18,7 +20,7 @@ process: process.cpp curved.o fcam/Demosaic.o fcam/Demosaic_ARM.o
 	$(CXX) $(CXXFLAGS) -Wall -O3 $^ -o $@ $(PNGFLAGS) -ldl -lpthread
 
 out.png: process
-	./process ../images/bayer_raw.png 3700 2.0 50 5 out.png
+	./process ../images/bayer_raw.png 3700 2.0 50 $(CAMERA_ITERATIONS) out.png
```

Comment:
The usage for ./process prints out "timing_iterations" for this argument: https://github.com/halide/Halide/blob/6783ce4d42615aa0ec7262919ed1b8042f23d3a7/apps/camera_pipe/process.cpp#L19

So I think this environment variable should be TIMING_ITERATIONS for consistency.